April 4 executive copy (#2607)
makerdao/community: governance/votes/Executive Vote - April 4, 2024.md

---

> title: Template - [Executive Vote] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions - April 4, 2024

Reviewer: We usually leave the spark proxy spell last, out of curiosity, why not this time?

Reviewer 2: In general we agree it should be last but this ordering matches the sheet and it was too late to change it at the point we were reviewing. In future Proxy Spell actions should go at the end of the copy - can you add this to the style guide?

summary: Increase the Governance Security Module Pause Delay, increase Spark MetaMorpho Maximum Debt Ceiling, add native vaults to Debt Ceiling Breaker, Trigger SparkLend Proxy Spell, and approve BT Project Ethena Resolutions. 
date: 2024-04-04T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions

Reviewer: ```suggestion
# [Executive Proposal] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions - April 4, 2024
```

The Governance Facilitators, SideStream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

> - The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.

Reviewer 2: ```suggestion
- The [Governance Security Module (GSM) Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.
```

> - The [Governance Security Module (GSM) Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.
> - Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased to **1 billion DAI**.
> - Native Vault Engine ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker) as detailed below.
> - A SparkLend Proxy Spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered.

Reviewer 2: ```suggestion
- The [Governance Security Module (GSM) Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be increased to **30 hours**.
- Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased to **1 billion DAI**.
- Native Vault Engine ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://mips.makerdao.com/mips/details/MIP113#10-2-3-debt-ceiling-breaker) as detailed below.
- A SparkLend Proxy Spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered.
```

### Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) Increase

- **Authorization:** [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/9)  
- **Proposal:** [Forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)

If this executive proposal passes, the Spark MetaMorpho Vault DDM Maximum Debt Ceiling (line) will increase by 900 million DAI, from 100 million DAI to **1 billion DAI**.

### Add ilks to [Debt Ceiling Breaker (LINE_MOM)](https://mips.makerdao.com/mips/details/MIP113#10-2-3-debt-ceiling-breaker)

- **Authorization:** [Poll](https://vote.makerdao.com/polling/QmZsAM36#vote-breakdown)  
- **Proposal:** [Forum post](https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal-gov10-2-3-1a-edit/24005)

If this executive proposal passes, the following ilks will be added to the Debt Ceiling Breaker (LINE_MOM):

* ETH-A
* ETH-B
* ETH-C
* WSTETH-A
* WSTETH-B
* WBTC-A
* WBTC-B
* WBTC-C

### Trigger SparkLend Proxy Spell

> - **Authorization:** Polls: [1](https://vote.makerdao.com/polling/QmdjqTvL), [2](https://vote.makerdao.com/polling/QmaEqEav), [3](https://vote.makerdao.com/polling/QmbCWUAP). [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)  

Reviewer: ```suggestion
- **Authorization:** Polls: [1](https://vote.makerdao.com/polling/QmdjqTvL), [2](https://vote.makerdao.com/polling/QmaEqEav), [3](https://vote.makerdao.com/polling/QmbCWUAP). [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/9)  
```

If this executive proposal passes, the SparkLend Proxy spell at [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be executed. The expected contents of this are:

#### Increase ETH Borrow Cap `gap` Parameter

Increases [ETH Market](https://app.spark.fi/reserve-overview/?underlyingAsset=0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2&marketName=proto_spark_v3) borrow cap `gap` parameter by 10,000 ETH from 10,000 ETH to **20,000 ETH**

#### Increase WBTC Supply Cap `max` Parameter

Increases [WBTC market](https://app.spark.fi/reserve-overview/?underlyingAsset=0x2260fac5e5542a773aa44fbcfedf7c193bc2c599&marketName=proto_spark_v3) supply cap `max` parameter by 4,000 WBTC from 6,000 WBTC to **10,000 WBTC**

#### Activate Kill Switch Security Module

[Adds a kill switch](https://github.com/marsfoundation/sparklend-kill-switch) to disable all borrowing across markets in the event of a depeg on key collateral assets.

#### Update MetaMorpho Market Caps

Updates Market Cap Parameters to the following:

> USDe 77% LLTV pool cap: **1 billion DAI**
> USDe 86% LLTV pool cap: **500 million DAI**
> USDe 91.5% LLTV pool cap: **200 million DAI**
> USDe 94.5% LLTV pool cap: **10 million DAI**
> sUSDe 77% LLTV pool cap: **1 billion DAI**
> sUSDe 86% LLTV pool cap: **200 million DAI**
> sUSDe 91.5% LLTV pool cap: **50 million DAI**
> sUSDe 94.5% LLTV pool cap: **10 million DAI**

Reviewer 2: :yellow_circle:  **medium**

This is all going on to one line when rendered in rich diff. The easiest fix is to make it bullet points.

```suggestion
* USDe 77% LLTV pool cap: **1 billion DAI**
* USDe 86% LLTV pool cap: **500 million DAI**
* USDe 91.5% LLTV pool cap: **200 million DAI**
* USDe 94.5% LLTV pool cap: **10 million DAI**
* sUSDe 77% LLTV pool cap: **1 billion DAI**
* sUSDe 86% LLTV pool cap: **200 million DAI**
* sUSDe 91.5% LLTV pool cap: **50 million DAI**
* sUSDe 94.5% LLTV pool cap: **10 million DAI**
```

### Approve TACO Resolutions

> - **Authorization:** $Missing? 

Reviewer 2: ```suggestion
- **Authorization:** [Forum post](https://forum.makerdao.com/t/bt-project-ethena-risk-legal-assessment/23978/9)
```

---

> title: Template - [Executive Vote] Increase GSM Pause, Increase Spark MetaMorpho Vault Debt Ceiling, Add Standard ilks to LINE_MOM, SparkLend Proxy Spell , and TACO Resolutions - April 4, 2024

Reviewer 2: :yellow_circle: **medium**

Improve accuracy and use human readable names

```suggestion
title: Template - [Executive Vote] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions - April 4, 2024
```

> summary: Increase the Governance Pause Delay, Increase Spark MetaMorph Maximum Debt Ceiling, Add non-PSM vaults to Debt Ceiling Breaker, Trigger SparkLendProxy Spell, and Approve BT Project Ethena Resolutions. 

Reviewer 2: :yellow_circle: **medium**

1. fix typos
2. remove capitalization
3. improve clarity

```suggestion
summary: Increase the Governance Security Module Pause Delay, increase Spark MetaMorpho Maximum Debt Ceiling, add native vaults to Debt Ceiling Breaker, Trigger SparkLend Proxy Spell, and approve BT Project Ethena Resolutions. 
```

date: 2024-04-04T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] Increase GSM Pause, Increase Spark MetaMorpho Vault Debt Ceiling, Add Standard ilks to LINE_MOM, TACO Resolutions, and SparkLend Proxy Spell - April 4, 2024

Reviewer 2: :yellow_circle: **medium**

see above

```suggestion
# [Executive Proposal] Increase GSM Pause Delay, Increase Spark MetaMorpho Vault Maximum Debt Ceiling, Add Native Vaults to Debt Ceiling Breaker, SparkLend Proxy Spell, and TACO Resolutions
```

The Governance Facilitators, SideStream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

> - The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **16 hours**.

Reviewer 2: :red_circle: **Blocking**

Incorrect value

```suggestion
- The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **30 hours**.
```

- Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased to **1 billion DAI**.

> - Standard ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker) as detailed below.

Reviewer 2: :green_circle: **minor**

```suggestion
- Native Vault Engine ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker) as detailed below.
```

> - The [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) will be increased to **16 hours**.
> - Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased to **1 billion DAI**.
> - Standard ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker) as detailed below.

Reviewer 2: :green_circle: **minor**

Link to Scope definitions where possible

```suggestion
- The [Governance Pause Delay (GSM)](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be increased to **16 hours**.
- Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) will be increased to **1 billion DAI**.
- Standard ilks will be added to the [Debt Ceiling Breaker (LINE_MOM)](https://mips.makerdao.com/mips/details/MIP113#10-2-3-debt-ceiling-breaker) as detailed below.
```

- A SparkLend Proxy Spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered.
- [A set of TACO Resolutions](https://gateway.pinata.cloud/ipfs/Qmf8Nv4HnTFNDwRgcLzRgBdtVsVVfKY2FppaBimLK9XhxB) for BT Project Ethena will be approved. 

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **16 hours**.

This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

> ### [Governance Pause Delay (GSM)](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) Increase

Reviewer 2: :green_circle: **minor**

See above

```suggestion
### [Governance Pause Delay (GSM)](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) Increase
```

- **Authorization:** [Poll](https://vote.makerdao.com/polling/QmcLsYwj)  
- **Proposal:** [Forum post](https://forum.makerdao.com/t/gsm-pause-delay-increase-proposal/23929)

If this executive proposal passes, The Governance Pause Delay (GSM) will be increased by 14 hours, from 16 to **30 hours**.

> ### Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increase

Reviewer 2: :green_circle: **minor**

See above

```suggestion
### Spark MetaMorpho Vault DDM [Maximum Debt Ceiling (line)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) Increase
```

- **Authorization:** [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006/9)  
- **Proposal:** [Forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)

> If this executive proposal passes, the Spark MetaMorpho Vault DDM Maximum Debt Ceiling (line) will increase by 900 million DAI, from 100 million to **1 billion DAI**.

Reviewer 2: :green_circle: **minor**

Add missing unit

```suggestion
If this executive proposal passes, the Spark MetaMorpho Vault DDM Maximum Debt Ceiling (line) will increase by 900 million DAI, from 100 million DAI to **1 billion DAI**.
```

> ### Add ilks to [Debt Ceiling Breaker (LINE_MOM)](https://manual.makerdao.com/governance-processes/executive-vote-considerations/gsm-exceptions#debt-ceiling-breaker)

Reviewer 2: :green_circle: **minor**

See above

```suggestion
### Add ilks to [Debt Ceiling Breaker (LINE_MOM)](https://mips.makerdao.com/mips/details/MIP113#10-2-3-debt-ceiling-breaker)
```

- **Authorization:** [Poll](https://vote.makerdao.com/polling/QmZsAM36#vote-breakdown)  
- **Proposal:** [Forum post](https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal-gov10-2-3-1a-edit/24005)

If this executive proposal passes, the following ilks will be added to the Debt Ceiling Breaker (LINE_MOM):

* ETH-A
* ETH-B
* ETH-C
* WSTETH-A
* WSTETH-B
* WBTC-A
* WBTC-B
* WBTC-C

### Trigger SparkLend Proxy Spell

- **Authorization:** Polls: [1](https://vote.makerdao.com/polling/QmdjqTvL), [2](https://vote.makerdao.com/polling/QmaEqEav), [3](https://vote.makerdao.com/polling/QmbCWUAP). [Forum Post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)  
- **Proposal:** Forum posts: [1](https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918), [2](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006)

> If this executive proposal passes, the SparkLend Proxy spell [0x7748C5E6EEda836247F2AfCd5a7c0dA3c5de9Da2](https://etherscan.io/address/0x7748c5e6eeda836247f2afcd5a7c0da3c5de9da2) will be triggered. In addition to polled changes on Activating the Kill Switch Security Module and increasing WBTC and ETH lending caps, this proxy spell contains changes suggested and approved in this [forum post](https://forum.makerdao.com/t/morpho-spark-dai-vault-update-1-april-2024/24006).

Reviewer 2: :yellow_circle: **Medium**

We strongly advise that Spark spell items are itemized for the benefit of voters like in previous docs.

See: https://github.com/makerdao/community/pull/2597/files#diff-46eac166654845e1c858fca919ba2d2ec7d5114fdc7248747924e2fc3b8b8e14R125-R127
and: https://github.com/makerdao/community/pull/2580/files#diff-a856ab68877ffd3cd18fa55b98aee86bfd37a7f0c1b7ce54bee1e43ee9634d00R102-R225

### Approve TACO Resolutions

> - **Authorization:** $Missing? 

Reviewer 2: :red_circle: **Blocking**

We understand this is pending and the Responsible Facilitator has been contacted.
